plugin/manager.go

```diff
@@ -141,6 +150,24 @@ func (pm *Manager) reload() error {
 				return
 			}
 
+			if p.Rootfs != "" {
+				p.Rootfs = filepath.Join(pm.libRoot, p.PluginObj.ID, "rootfs")
+			}
+
+			// We should only enable rootfs propagation for certain plugin types that need it.
+			for _, typ := range p.PluginObj.Config.Interface.Types {
+				if typ.Capability == "volumedriver" && typ.Prefix == "docker" && strings.HasPrefix(typ.Version, "1.") {
+					if p.PluginObj.Config.PropagatedMount != "" {
+						// TODO: sanitize PropagatedMount and prevent breakout
+						p.PropagatedMount = filepath.Join(p.LibRoot, p.PluginObj.Config.PropagatedMount)
```

Comment:
Yes. Having the same dir as the mount in the config could be less confusing. On the other hand, having a standard dir such as `mnt` is also reasonable. I'll let @tiborvass decide. Also, we dont have to do this in this PR.